presto-product-tests-launcher/src/main/resources/docker/presto-product-tests/conf/environment/singlenode-kerberos-hdfs-impersonation-with-data-protection/hive.properties

```diff
@@ -0,0 +1,19 @@
+connector.name=hive-hadoop2
+hive.metastore.uri=thrift://hadoop-master:9083
+hive.config.resources=/docker/presto-product-tests/conf/presto/etc/hive-default-fs-site.xml,/docker/presto-product-tests/conf/presto/etc/hive-data-protection.xml
```

Comment:
a TODO not to require `hive-data-protection.xml`?

(also you may want to follow `*-site.xml` convention, as in `hive-default-fs-site.xml`)

Reply:
> a TODO not to require hive-data-protection.xml?

Not sure it is worth it. It looks like we should map the two properties which we have in .xml 1-1 to Presto properties. It can be done, but I am not 100% those are commonly use enough to justify that. I actually have code for one of those, already, but I shelved it to discuss if we really want it.

Reply:
Fair